tests/integration/test_valid_data_in_request.py

```diff
@@ -97,3 +98,17 @@ def test_various_packages(test_env):
         utils.assert_properly_completed_response(completed_response)
 
         assert len(completed_response.data["dependencies"]) == package["dependencies_count"]
+
+
+def test_utc_datetime(test_env):
+    env_data = utils.load_test_data("pip_packages.yaml")["local_path"]
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
+    expected = datetime.datetime.utcnow()
```

Comment:
Minor point, but we might want to change the names of the actual/expected variables to something more meaningful. Consider something like utc_time and request_created_time. What do you think?

Reply:
Got it. Added the changes.